plugins/com.google.cloud.tools.eclipse.appengine.libraries/schema/com.google.cloud.tools.eclipse.appengine.libraries.exsd

```diff
@@ -0,0 +1,194 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="com.google.cloud.tools.eclipse.appengine.libraries" xmlns="http://www.w3.org/2001/XMLSchema">
```

Comment:
Looking into this further, plugin.xml never seems to use anything except the default namespace. Is that correct? 

If so, using any targetNamespace here is simply incorrect and if the other document does it, I'm 70% sure it's wrong (not 90% because the schema spec is ridiculously complex). If something in Eclipse expects this to happen, then that something is in violation of the schema spec.